app/Http/Requests/StoreUserInvitation.php

```diff
@@ -24,12 +25,21 @@ public function authorize()
     public function rules()
     {
         return [
-            'firstname'        => 'required|string|max:255',
-            'lastname'         => 'required|string|max:255',
-            'email'            => 'required|email|max:255|unique:users,email',
-            'username'         => 'required|string|max:255|unique:users,username',
-            'password'         => 'required|string|max:255',
-            'invitation_token' => 'required|uuid|max:255'
+            'firstname'             => 'required|string|max:255',
+            'lastname'              => 'required|string|max:255',
+            'email'                 => 'required|email|max:255|unique:users,email',
+            'username'              => 'required|string|max:255|unique:users,username',
+            'password'              => ['required','string', 'confirmed','regex:/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*(_|[^\w])).+$/','min:8','max:20'],
```

Comment:
Create a rule https://laravel.com/docs/7.x/validation#custom-validation-rules for the password requirements, to keep the code DRY

Reply:
https://github.com/THM-Health/PILOS/pull/23/commits/1a33a7aa89dddde61afd7c0994421ca8d1e20bfb added UserPassword rule